x86_symbolic/src/Data/Macaw/X86/Symbolic.hs

```diff
@@ -472,6 +480,102 @@ instance GenArchInfo LLVMMemory M.X86_64 where
     , updateReg = x86UpdateReg
     }
 
+-- | Helper, not exported
+writeBytesBackwards ::
```

Comment:
I think it would be helpful to state in the comments that:

1. This writes each byte in `bytes` at successively smaller negative offsets from `ptr`.
2. This assumes the precondition that `ptr`'s allocation is large enough to store the data in `bytes`.

Reply:
I've removed this function, will add haddocks to functions that make assertions